packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -140,7 +140,7 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
          * and apply actions only to them
          */
         commands.registerCommand({ id: 'workbench.action.files.newUntitledFile' }, {
-            execute: () => open(this.openerService, createUntitledResource().uri)
+            execute: async () => open(this.openerService, (await createUntitledResource()).uri)
```

Comment:
We should extract a function which creates untitled URI instead there is not reason to create a full resource.

Reply:
fixed, done